docker/protect.cwl

```diff
@@ -63,49 +63,49 @@ hints:
 
 inputs:
 
-  sample-name:
+  sample_name:
```

Comment:
I switched to using underscores to separate the cwl identifiers from the ProTECT switch names which us hyphens. There were a couple of cwl identifiers that had hyphens and I changed them to underscores to make everything uniform